### PR TITLE
Add Wandera (and a few other merged changes)

### PR DESCRIPTION
**Pull request by mattvlas**
Adds or updates the following events or companies:
 - 
 - 

### Company updates
 - [ ] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's website (Please put N/A if there is no public post)

### Event updates
 - [ ] I have linked to the article about the change, not the event's website

### CODE REVIEW

File: _data/companies.yml [hunk 1]
```diff
@@ -306,13 +306,6 @@
   events: Restricted
   last_update: 2020-03-12
 
-- name: CERN [[1]](https://hse.cern/news-article/coronavirus-information-measures-and-recommendations)
-  wfh: Planned
-  travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-10
-
```
Comment (drewbrew) [inline _data/companies.yml]: It looks like this PR is trying to overwrite some recent changes. Would you mind undoing this?

File: _data/companies.yml [hunk 2]
```diff
@@ -1504,11 +1497,11 @@
...
 - name: Slido
-  wfh: Required
+  wfh: Encouraged
   travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
+  visitors: "?"
+  events: "?"
+  last_update: 2020-03-11
```
Comment (drewbrew) [inline _data/companies.yml]: This too

File: _data/companies.yml [hunk 3]
```diff
@@ -1803,6 +1796,13 @@
   visitors: "?"
   events: "?"
   last_update: 2020-03-06
+  
+  - name: Wandera
```
Comment (drewbrew) [inline _data/companies.yml]:
```suggestion
- name: Wandera
```

**Comment by mattvlas** [comment]
Not sure how I managed to do this. Will close and re-try.

**Pull request closed without merging**